scss/common/_forms.scss

```diff
@@ -58,6 +58,7 @@
         border-style: solid;
         box-sizing: border-box;
         position: relative;
+        transition: all .1s ease; // sass-lint:disable-block no-transition-all
```

Comment:
Do we really need to transition all properties? It seems like `transition: color .1s ease, background .1s ease;` will be sufficient.